ua/org.eclipse.help.base/src/org/eclipse/help/internal/base/HelpBasePlugin.java

```diff
@@ -71,6 +71,16 @@ public String getDocumentMessage(boolean embedded) {
 		public boolean getDocumentMessageUsesLiveHelp(boolean embedded) {
 			return false;
 		}
+
+		@Override
+		public boolean isSyncSelection() {
+			return false;
```

Comment:
Default should be "true" because it is our "legacy" setting. However, this can be removed here  if interface would provide default method